API/Controllers/SeriesController.cs

```diff
@@ -148,7 +148,7 @@ public async Task<ActionResult> UpdateSeries(UpdateSeriesDto updateSeries)
         }
 
         [HttpGet("recently-added")]
-        public async Task<ActionResult<IEnumerable<SeriesDto>>> GetRecentlyAdded(int libraryId = 0, int limit = 20)
+        public async Task<ActionResult<IEnumerable<SeriesDto>>> GetRecentlyAdded(int limit, int libraryId = 0)
```

Comment:
We don't need to add a new API, but instead can just pass [FromQuery] UserParams userParams. This is where I store Pagination information, like in other methods here (see Collections or Series).